run-tests

```diff
@@ -4,9 +4,17 @@ FWDIR="$(cd "`dirname $0`"; pwd)"
 cd "$FWDIR"
 
 set -x
+# Attempt to find SPARK_HOME
+if [ -z "$SPARK_HOME" ]; then
+  if hash find_spark_home.py 2>/dev/null; then
```

Comment:
What is this `find_spark_home.py` script, it's not described anywhere in this repo.

Reply:
this is a "findspark" lib in setup.py. It was inherited from the source 'https://github.com/holdenk/spark-testing-base' repos